### DISCUSSION TITLE
Read articles should not be deleted automatically

### DISCUSSION THREAD

**Discussion by borges-filipe** [spacecowboy/Feeder#192] [Ideas]
All read articles are currently deleted upon leaving the respective feed. If this is the expected behaviour, there should be an option in settings to keep read articles for a set amount of time.

I often like to reread articles and do not necessarily want to add them to saved. Ideally read articles would be grayed out (or marked in some other way) instead of deleted.

**Comment by borges-filipe** [comment]
Nevermind, I have now found the option to change behaviour. My first instinct had been to look in settings

**Comment by spacecowboy** [comment]
For context, in case anyone finds this with the same question, the solution is to change your filters to show read items
